src/Analyzer.PowerShellRuleEngine/PowerShellRuleEngine.cs

```diff
@@ -34,20 +34,27 @@ public PowerShellRuleEngine()
         {
             this.PowerShell = System.Management.Automation.PowerShell.Create();
 
-#if (DEBUG)
-            PowerShell.Commands.AddCommand("Set-ExecutionPolicy")
-                .AddParameter("Scope", "Process") // Affects only the current PowerShell session
-                .AddParameter("ExecutionPolicy", "Unrestricted");
-            PowerShell.AddStatement();
-#endif
-
             PowerShell.Commands.AddCommand("Import-Module")
                 .AddParameter("Name", Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + @"\TTK\arm-ttk.psd1"); // arm-ttk is added to the needed project's bins directories in build time 
             PowerShell.AddStatement();
 
             PowerShell.Invoke();
         }
 
+        /// <summary>
+        /// Sets the PowerShell execution policy for the current process as unrestricted
+        /// </summary>
+        public static void SetExecutionPolicy()
+        {
+            var powerShell = System.Management.Automation.PowerShell.Create();
+
+            powerShell.Commands.AddCommand("Set-ExecutionPolicy")
+                .AddParameter("Scope", "Process") // Affects only the current PowerShell session
+                .AddParameter("ExecutionPolicy", "Unrestricted");
+
+            powerShell.Invoke();
```

Comment:
Can this code go into the unit test itself?

Reply:
We need to do it for two unit test projects, apparently those are executed in different processes. I thought about creating a special PowerShell utility class, or something like that, but this looked better to me, what do you think?

Reply:
Since it's only needed for tests, it should stay in the test projects if possible.  I think it's fine to have it duplicated within the tests for simplicity.

Reply:
But it will be needed in the engine when we support custom PowerShell rules, we want to duplicate code anyways?

Reply:
We still need to finish a design for that scenario first.